request.go

```diff
@@ -117,6 +117,19 @@ func (r *Request) SetHeaders(headers map[string]string) *Request {
 	return r
 }
 
+// SetHeaderVerbatim method is to set a single header field and its value verbatim in the current request.
+//
+// For Example: To set `all_lowercase` and `UPPERCASE` as `available`.
+// 		client.R().
+//			SetHeader("all_lowercase", "available").
+//			SetHeader("UPPERCASE", "available")
+//
+// Also you can override header value, which was set at client instance level.
+func (r *Request) SetHeaderVerbatim(header, value string) *Request {
```

Comment:
@Lao-Tuo Please add the same nature of the method into the Resty client struct too. Since resty provides most methods at both client and request level.